src/app/components/question-approval-page/question-approval-page.component.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { QuestionServiceStub } from 'src/testing/question.service.stub';
+import { QuestionApprovalPageComponent } from './question-approval-page.component';
+import { ClarityModule } from '@clr/angular';
+import { ReactiveFormsModule } from '@angular/forms';
+import { QuestionService } from 'src/app/services/question.service';
+import { KatexModule } from 'ng-katex';
+
+describe('QuestionApprovalPageComponent', () => {
+    let component: QuestionApprovalPageComponent;
+    let fixture: ComponentFixture<QuestionApprovalPageComponent>;
+
+    beforeEach(async(() => {
+      TestBed.configureTestingModule({
+        declarations: [QuestionApprovalPageComponent],
+        imports: [ClarityModule,
+          ReactiveFormsModule,
+          KatexModule]
+      }).overrideComponent(QuestionApprovalPageComponent, {
+        set: {
+          providers: [
+            { provide: QuestionService, useValue: QuestionServiceStub }
+          ]
+        }
+      })
+        .compileComponents();
+    }));
+
+    beforeEach(() => {
+      fixture = TestBed.createComponent(QuestionApprovalPageComponent);
+      component = fixture.componentInstance;
+      fixture.detectChanges();
+    });
+
+    it('should create', () => {
```

Comment:
An important test you might want to consider is what happens if the questionQueue doesn't return a "good" response. How does this page handle it? Reason being that now, this is what i'm seeing:
![image](https://user-images.githubusercontent.com/20339399/88507594-a961d980-cfd4-11ea-933b-69523f7c0f9d.png)

Seems like it tries to reach out to master instead of the ft1 version?